src/assets/wise5/components/table/table-student/table-student.component.ts

```diff
@@ -1010,6 +1013,22 @@ export class TableStudent extends ComponentStudent {
     });
   }
 
+  updateColumnsUsed(): void {
+    const firstRow = this.tableData[0];
+    for (let c = 0; c < firstRow.length; c++) {
+      this.columnIndexToIsUsed.set(c, this.isColumnUsed(c));
+    }
+  }
+
+  isColumnUsed(columnIndex: number): boolean {
+    return (
+      columnIndex === this.dataExplorerXColumn ||
+      this.dataExplorerSeries.some((series) => {
+        return columnIndex === series.yColumn;
+      })
```

Comment:
In this block, should the operands be switched, since series.yColumn changes and columnIndex is fixed?
```suggestion
      this.dataExplorerSeries.some((series) => {
        return series.yColumn === columnIndex;
      })
```